### PR TITLE
[fixcommonerrors] - add reset default button.

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

**Comment by darnn** [comment]
This looks good! Will there be a way to set the default?

**Comment by ivandrofly** [comment]
> This looks good! Will there be a way to set the default?

not yet, probably in future it can be implemented 😉 

pst: actually it would be better to have groups just like multiple-replace...

File: src/Forms/FixCommonErrors.cs [hunk 1]
```diff
@@ -1608,5 +1609,10 @@ private void comboBoxLanguage_Enter(object sender, EventArgs e)
             SaveConfiguration();
         }
 
+        private void buttonResetDefault_Click(object sender, EventArgs e)
+        {
+            Configuration.Settings.CommonErrors.LoadDeafultConfigs();
```
Comment (ivandrofly) [inline src/Forms/FixCommonErrors.cs]: oops... typo **LoadDeafultConfigs**

**Comment by niksedk** [comment]
Could you move the new button to the right of "Invert selection"?
And perhaps call it "Select default" or "Select default fixes" or similar...

Also, don't know if the default settings are good - don't reset my settings often ;)

**Comment by ivandrofly** [comment]
Updated @niksedk

**Pull request merged**